packages/editor/src/client/atoms/descriptionList.tsx

```diff
@@ -44,7 +45,16 @@ export function DescriptionListItem({label, children}: DescriptionListItemProps)
         style={{
           marginLeft: 20
         }}>
-        {children}
+        {children || (
+          <div
+            style={{
+              borderRadius: '8px',
+              padding: '6px',
+              backgroundColor: '#FFEBCD'
+            }}>
+            {message}
+          </div>
+        )}
```

Comment:
@camilleory From line 48 - 57 May be it can be structured as follow
{message 
  ?     (<div
            style={{
              borderRadius: '8px',
              padding: '6px',
              backgroundColor: '#FFEBCD'
          }}>
            {message}
          </div>) 
     : (<span>{children}<span>)}